witchcraft-server/src/logging/mod.rs

```diff
@@ -59,10 +64,43 @@ pub(crate) async fn init(
     let request_logger = logger::appender(install, metrics, hooks).await?;
     let audit_logger = logger::appender(install, metrics, hooks).await?;
 
+    // ignore if the audit logger is already filled
+    let _ = AUDIT_LOGGER.fill(audit_logger.clone());
```

Comment:
We should update the audit logger service to use the new static methods so we don't need to clone the appender and use 2 copies.

Reply:
The main issue there is that it makes it kind of a pain to test the audit logger service. I think the clone that takes place is just incrementing a ref count under the hood which should be pretty cheap?

Reply:
Oh right. The Clone impl isn't correct though - it'll cause bad behavior when interacting with the Drop implementation.

Reply:
Its sink etc implementations also assume there's only one user at a time.

Reply:
I updated to just do the `start_close` in the drop implementation, but not sure if that is the right way to go about it? We could also just wrap the top level logger in an `Arc`

Reply:
It can't go in State's drop impl since that is also held onto by the writing task. I think we'll need to work with an `Arc<Mutex<AsyncAppender>>` if we want to have two handles.

Reply:
Ah okay, I think it should be updated with the latest commit to just have the audit logger be an arc mutex? It's a little bit ugly but should work for now